test/models/setting_test.rb

```diff
@@ -204,11 +204,12 @@ def test_value_is_updated_after_change_in_SETTINGS
     setting_name = "foo_#{rand(1000000)}"
     Setting.create!(:name => setting_name, :value => "bar", :default => "default", :description => "foo")
 
-    SETTINGS.stubs(:key?).with(setting_name.to_sym).returns(true)
-    SETTINGS.stubs(:[]).with(setting_name.to_sym).returns("no-bar")
+    SETTINGS[setting_name.to_sym] = "no-bar"
```

Comment:
These stubs failed because creating the setting checked if taxonomies are enabled for the audit, which the stub didn't expect. However, `SETTINGS` is just a hash, so stubbing it is actually slower then just updating it.